frontend/src/common/queries/filter.ts

```diff
@@ -377,6 +383,40 @@ function buildDatasetRow(
   return sortCategoryValues(datasetRow);
 }
 
+/**
+ * Build summary citation format from given publisher metadata:
+ * Last name of first author (publication year) journal abbreviation such as Ren et al. (2021) Cell.
+ * @param publisherMetadata - Publication metadata of a collection.
+ */
+export function buildSummaryCitation(
+  publisherMetadata?: PublisherMetadata
+): string {
+  if (!publisherMetadata) {
+    return "";
+  }
+  const { authors, journal, published_year: publishedYear } = publisherMetadata;
+  const [firstAuthor] = authors;
```

Comment:
just curious, is it possible that `firstAuthor` could be `undefined`? If so `firstAuthor.name` would throw a runtime error 😄 !

Reply:
Good catch, Timmy! Let me add some handling just in case.